Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/SharedLogic/Analytics.hs

```diff
@@ -504,6 +750,42 @@ handleCacheMissForOperatorAnalyticsAllTime operatorId allTimeKeysData = do
           logTagInfo "OperatorAnalyticsAllTime" $ "inProgress lock already held for operatorId: " <> operatorId <> " (race detected in else). Waiting for it to clear."
           handleCacheMissForOperatorAnalyticsAllTime operatorId allTimeKeysData
 
+handleCacheMissForFleetAnalyticsAllTime ::
+  ( MonadFlow m,
+    EsqDBFlow m r,
+    CacheFlow m r,
+    Redis.HedisFlow m r,
+    HasField "serviceClickhouseCfg" r CH.ClickhouseCfg,
+    HasField "serviceClickhouseEnv" r CH.ClickhouseEnv
+  ) =>
+  Text ->
+  [Text] ->
+  m FleetAllTimeFallbackRes
+handleCacheMissForFleetAnalyticsAllTime fleetOwnerId fleetAllTimeKeysData = do
```

Comment:
Also duplication between `handleCacheMissForOperatorAnalyticsAllTime` and `handleCacheMissForFleetAnalyticsAllTime`